contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -5984,8 +5984,13 @@ default_collation_for_builtin_type(Type typ, bool handle_pg_type)
 	 * Special handling for PG datatypes such as TEXT because Babelfish does not define sys.TEXT.
 	 * This is required as Babelfish currently does not handle collation of String Const node correctly.
 	 * TODO: Fix the handling of the collation for String Const node.
+	 * 
+	 * When pltsql_psql_logical_babelfish_db_name is set, return CLUSTER_COLLATION_OID() for TEXT (e.g: string literals)
```

Comment:
also mention that psql_logical_babelfish_db_name can only be set inside pg dialect.

Reply:
updated comment in latest rev